### PR TITLE
[MM-33080] Set status after installation

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
I've checked the code base and this seems like the only place that was missing the status.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-33080

### CODE REVIEW

File: server/api/impl/admin/provision.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ func (adm *Admin) ProvisionApp(cc *apps.Context, sessionToken apps.SessionToken,
 
 	app := &apps.App{
 		Manifest:       manifest,
+		Status:         apps.AppStatusInstalled,
```
Comment (catalintomai) [inline server/api/impl/admin/provision.go:39]: Why do we set the `AppStatusInstalled` on the `ProvisionApp` call (as opposed to the `InstallApp` call)?

Comment (hanzei) [reply server/api/impl/admin/provision.go:39]: Good point. Maybe I've misunderstood what `ProvisionApp` does. Let me double check and come back to you.

**Comment by levb** [comment]
https://github.com/mattermost/mattermost-plugin-apps/pull/85 makes this PR redundant

**Comment by hanzei** [comment]
Yep, let's close this one.

**Pull request closed without merging**